src/scrabble.js

```diff
@@ -2,31 +2,80 @@ const { letters } = require('./letter.js')
 
 const scrabble = (word) => {
   let output = 0
-  const scrabbleArr = []
+  let mulitplier = 1
+  // Arrays to test for valid input
+  const checkerArr = []
+  const checkerArr2 = []
 
+  // Checking for any number in input
   function checkString(string) {
     return /^[0-9]*$/.test(string)
   }
 
   const stringContainsNum = checkString(word)
 
-  if (typeof word === 'string' && stringContainsNum === false) {
+  // Multiple checks for valid input
+  if (
+    typeof word === 'string' &&
+    stringContainsNum === false &&
+    /^[a-zA-Z{}[\]]*$/.test(word)
+  ) {
     for (let i = 0; i < word.length; i++) {
-      scrabbleArr.push(word[i].toLowerCase())
+      const letter = word[i].toLowerCase()
+
+      // Checking for ({,[), if true then multiplier applied
```

Comment:
I like how you've solved the extension - I'd never have thought to include a clause to reset the multiplier to 1, that seems like a really smart way to go about this!

Reply:
Thanks :)